core/tests/protractor_utils/users.js

```diff
@@ -72,8 +76,10 @@ var login = async function(
 
 var logout = async function() {
   var driver = browser.driver;
+  browser.waitForAngularEnabled(false);
```

Comment:
Why do we need this?

Reply:
If i dont use this, test fails with following error:
![Screenshot from 2021-02-13 12-02-30](https://user-images.githubusercontent.com/24855641/107843735-76a5f400-6df3-11eb-93df-282fa3b82ebb.png)
![Screenshot from 2021-02-13 12-02-44](https://user-images.githubusercontent.com/24855641/107843740-79084e00-6df3-11eb-8c57-32c825ae4113.png)